README.md

```diff
@@ -25,16 +25,10 @@ This web app, helps you keep track of your budget. You can add categories and de
 
 To get a local copy up and running follow these simple example steps.
 
-- Copy Repo Link
-
-git@github.com:gracekabaghe/budget-app.git
-```
-- Get the directory that you want to clone the repository.
-
-- Open the command prompt in this directory.
-
 - Copy clone of the Repo
```

Comment:
- [ ] The back button for the categories is not working as expected. Kindly fix it.

![image](https://user-images.githubusercontent.com/55635977/159937811-6aa2c8bb-54db-429f-ab73-7d2f8b429e11.png)

Reply:
Fixed